portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,21 +26,50 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
+import java.util.ArrayList; 
+import java.util.List;
+import com.google.gson.Gson;
+import com.google.sps.data.Task;
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-/**
+  private int quantity = 5; 
```

Comment:
I think you can provide a better UX if you store quantity only on the client side. 

Let the javascript populate the GET request with a URL parameter containing the quantity that the user selected in the UI.